sale_report_margin/__manifest__.py

```diff
@@ -2,8 +2,9 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Sale Report Margin",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "summary": "Adds purchase_price field to sale report",
```

Comment:
This is more a technical summary than a functional one (should be understandable by everyone.

Reply:
Updated and rebased too. Hope it's more understandable now.